dll/win32/kernel32/winnls/nls/afk.nls

```diff
@@ -18,13 +18,11 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
-#pragma code_page(1252)
-
-LANGUAGE LANG_AFRIKAANS, SUBLANG_DEFAULT
-LOCALE_FONTSIGNATURE RCDATA { L"\x002f\x8000\x3808\x0000\x0000\x0000\x0000\x0000\x0001\x0000\x0000\x4000\x00ff\x003f\x0000\xffff" }
+#pragma code_page(65001) /* UTF-8 */
```

Comment:
You may as well remove all these pragmas from these files, and "factor it out" into the main locale_rc.rc, as we do in our other translations.

Reply:
I just copy-pasted triste from wine and see no reason to introduce differences

Reply:
ok so let's keep these like that.